src/Microsoft.FSharp.Collections.ListModule.erl

```diff
@@ -1,54 +1,476 @@
 -module('Microsoft.FSharp.Collections.ListModule').
--compile(export_all).
+
+-export(
+    [
+        append/2,
+        average/1,
+        averageBy/2,
+        choose/2,
+        chunkBySize/2,
+        collect/2,
+        comparewith/3,
+        concat/1,
+        contains/2,
+        countBy/2,
+        distinct/1,
+        empty/0,
+        exists/2,
+        exists2/3,
+        filter/2,
+        find/2,
+        findIndex/2,
+        fold/3,
+        fold2/4,
+        foldBack/3,
+        foldBack2/4,
+        forall/2,
+        forall2/3,
+        head/1,
+        init/2,
+        isEmpty/1,
+        iter/2,
+        iter2/3,
+        iteri/2,
+        iteri2/3,
+        length/1,
+        map/2,
+        map2/3,
+        map3/4,
+        mapi/2,
+        mapi2/3,
+        max/1,
+        maxBy/2,
+        min/1,
+        minBy/2,
+        nth/2,
+        % ofArray/1,
+        % ofSeq/1,
```

Comment:
`SeqModule` is already partially implemented and the two functions we talk about here are available right now. The `seq()` type is opaque (and may change) and should only be known to the SeqModule itself so delegating to this module is the appropriate thing to do here in my opinion.